website/src/app/pages/data-distributions/charts/pie-chart.vl.ts

```diff
@@ -0,0 +1,166 @@
+import { VisualizationSpec } from 'vega-embed';
+import { VariableData, DistributionData } from '../data-distributions.component';
+
+export function createPieSpec(variable: VariableData, distributionData: DistributionData[] = []): VisualizationSpec {
+  return {
+    $schema: 'https://vega.github.io/schema/vega-lite/v5.json',
+    datasets: {
+      distribution: distributionData
+    },
+    height: 300,
+    data: {
+      name: 'distribution'
+    },
+    view: {
+      strokeOpacity: 0
+    },
+    transform: [
+      {
+        calculate: 'datum.value === 1 ? "True" : "False"',
```

Comment:
Leave True/False labels to the business logic. Just use whatever values are given. This will make all three charts act the same/be at the same level of specificity.